File: Java Script - Introduction to Programming/3.2. Simple Conditions - Exam Problems/03. Sleepy Tom Cat.js
function sleepyTomCat(days) {
  let gameTomMin = 30000
  let ownerWorking = 63
  let ownerOff = 127

  let year = (365 - days)
  let yearToMin = (year * ownerWorking) + (days * ownerOff)
  let subtotal = Math.abs(gameTomMin - yearToMin)
  let hours = (Math.trunc(subtotal / 60))
  let min = subtotal % 60

  if (gameTomMin >= yearToMin) 
    return `Tom sleeps well\n${hours} hours and ${min} minutes less for play`
  return `Tom will run away\n${hours} hours and ${min} minutes more for play`
}

//console.log(sleepyTomCat(20))

File: Pyrhon - Introduction to Programming/3.2. Simple Conditions - Exam Problems/03. Sleepy Tom Cat.py
from math import trunc

days = int(input())
game_Tom_min = 30000
owner_working = 63
owner_Off = 127

year = (365 - days)
year_to_min = (year * owner_working) + (days * owner_Off)
subtotal = abs(game_Tom_min - year_to_min)
hours = (trunc(subtotal / 60))
minutes = subtotal % 60

if game_Tom_min >= year_to_min:
    print(f'Tom sleeps well\n{hours} hours and {minutes} minutes less for play')
else:
    print(f'Tom will run away\n{hours} hours and {minutes} minutes more for play')


#20
